juju/juju
# feat: use controller UUID in access perms

The controller access permission used the controller namespace
"controller" as the target of the permission. For all other permissions
the UUID of the target entity is used. It should be the same here.

The controller access permission is changed to use the controller UUID.
This has required piping the UUID through all the appropriate places.




## Checklist




## QA steps
```
juju bootstrap lxd test-user
juju add-model default
juju change-user-password admin
juju add-user alastair
juju change-user-password alastair
juju logout
juju login -u alastair -c test-user
```
Should now be able to access model.

## Documentation changes



## Links



**Jira card:** [JUJU-6260](https://warthogs.atlassian.net/browse/JUJU-6260)



[JUJU-6260]: https://warthogs.atlassian.net/browse/JUJU-6260?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
